lib/headers.js

```diff
@@ -102,7 +102,9 @@ class Headers extends AbstractHeaders {
         for (const name of Object.keys(riffHeaders)) {
             if (riffHeaders[name] && riffHeaders[name].values) {
                 const values = riffHeaders[name].values;
-                headers = headers.addHeader(name, ...values);
+                if (values != null && typeof values[Symbol.iterator] === 'function') {
```

Comment:
is this change related to the CI changes?

Reply:
yes, node 10 errs if you spread a non-iterable, while node 8 ignored it